x-pack/plugins/lens/public/visualizations/tagcloud/tags_dimension_editor.tsx

```diff
@@ -28,11 +28,10 @@ import {
   EuiBadge,
 } from '@elastic/eui';
 import { useState, MutableRefObject, useCallback } from 'react';
-import { PalettePicker } from '@kbn/coloring/src/shared_components/coloring/palette_picker';
 import { useDebouncedValue } from '@kbn/visualization-ui-components';
 import { getColorCategories } from '@kbn/chart-expressions-common';
 import type { TagcloudState } from './types';
-import { PalettePanelContainer } from '../../shared_components';
+import { PalettePanelContainer, PalettePicker } from '../../shared_components';
```

Comment:
ℹ️ Irrelevant with this change but it solves a problem with tagcloud defaulting to positive palette in the editor while the default is selected (exists in main)

Reply:
Tested. Looks good.